charon/src/lib.rs

```diff
@@ -24,6 +24,7 @@
 // For when we use charon on itself :3
 #![register_tool(charon)]
 
+extern crate rustc_apfloat;
```

Comment:
The main charon library should use the strict minimum amount of rustc dependencies, in the end I'd like to have 0 rustc dependencies, so I'm not a fan of adding one.

Reply:
Even better: we could use `rustv_apfloat` from `crates.io` here, since we don't need to pass the resulting values to rustc

Reply:
Done in https://github.com/AeneasVerif/charon/pull/348/commits/1bf4d29b4d8183b33cf881ae192358ac0dc4e399